### PR TITLE
[DOCS] Rename X-Pack Commands section

### PR DESCRIPTION
**Pull request by lcawl**
Renames the "X-Pack Commands" section such that it can contain reference information for other commands too. 

Related to  https://github.com/elastic/elasticsearch/pull/32281/files/5f6b084324c607a92020909df6730c8e8a4b10f6#r211410593

### CODE REVIEW

File: docs/reference/commands/index.asciidoc [hunk 1]
```diff
@@ -1,26 +1,25 @@
...
+{es} includes the following commands that help you configure security:
 
 * <<certgen>>
 * <<certutil>>
 * <<migrate-tool>>
-* <<saml-metadata>>
 * <<setup-passwords>>
 * <<syskeygen>>
+* <<saml-metadata>>
 * <<users-command>>
```
Comment (debadair) [inline docs/reference/commands/index.asciidoc]: Was this change intentional? I'd be inclined to list the items alphabetically as it's easier to maintain sanity when items are added.

**Review by debadair — CHANGES_REQUESTED** [review]
A few comments. I'm about to chat with Anne & will see if she has an opinion re "-tool".

File: docs/reference/commands/index.asciidoc [hunk 2]
```diff
@@ -1,26 +1,25 @@
...
-[[xpack-commands]]
-= {xpack} Commands
+[[commands]]
+= Commands
 
 [partintro]
 --
 
-{xpack} includes commands that help you configure security:
+{es} includes the following commands that help you configure security:
```
[the remark this answers is not in the document]
Comment (lcawl) [reply docs/reference/commands/index.asciidoc]: I was thinking we could add another section or expand the description when we knew what would be added here.

File: docs/reference/commands/index.asciidoc (hunk 1 above)
Comment (lcawl) [reply docs/reference/commands/index.asciidoc]: It wasn't alphabetical anymore, since those commands are now prefixed with "elasticsearch-"

File: docs/reference/commands/index.asciidoc [hunk 3]
```diff
@@ -1,26 +1,25 @@
-[role="xpack"]
-[[xpack-commands]]
-= {xpack} Commands
+[[commands]]
+= Commands
```
[the remark this answers is not in the document]
Comment (lcawl) [reply docs/reference/commands/index.asciidoc]: You're right, other than elasticsearch-translog, the other "command" hits in my search results are all APIs.

[the remark this answers is not in the document]
Comment (lcawl) [reply docs/reference/commands/index.asciidoc]: If you look in elasticsearch/bin, there are more, such as elasticsearch-croneval, elasticsearch-cli, elasticsearch-sql-cli, elasticsearch-env, etc
